index3.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <!-- CSS only -->
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" crossorigin="anonymous">
+  <title>Document</title>
+</head>
+<body>
+ </body>
+<center>
+ <div class="row text-center" style="width:86%">
+    
+    <div class="card-body bg-black">
+      <h1 class="card-title">LEADERBOARD</h4>
+      
+    </div>
+ 
+   
+  </div></center>
+<BR>
+    <BR>
+        <BR>
+<div class="container">
+    <div class="row text-center">
+
+        <!-- Team item -->
+        <div class="col-xl-3 col-sm-6 mb-5">
+            <div class="bg rounded shadow-sm py-5 px-4"><img src="https://www.biography.com/.image/ar_1:1%2Cc_fill%2Ccs_srgb%2Cfl_progressive%2Cq_auto:good%2Cw_1200/MTc5OTk2ODUyMTMxNzM0ODcy/gettyimages-1229892983-square.jpg" alt="" width="90" class="img-fluid rounded-circle mb-2 bg-secondary img-thumbnail shadow-sm">
+                <h6 class="mb-0">ELON MUSK</h5>
+             
+                <div class="col-xl-3 col-sm-6 mb-5"></div>  
+                <div class="bgm rounded shadow-sm py-5 px-4"> 
+
+                   <b> <p class="card-text">GITHUB ID:

```

Comment:
GitHub Username would be more suitable on the top, along with the Name.
Avoid using `<center>`. Maximize the use of Bootstrap classes and use explicit CSS only if necessary.